tool/tctl/common/app_command.go

```diff
@@ -104,7 +105,7 @@ func (c *AppsCommand) ListApps(clt auth.ClientI) error {
 			return trace.Wrap(err)
 		}
 	case err != nil:
-		return trace.Wrap(err)
+		return utils.FatalPredicateError(err)
```

Comment:
Hrmm, I think this is a bit confusing without looking at the implementation of `FatalPredicateError` first. Just reading this my first thought was why are all errors from `GetResourcesWithFilters` a fatal predicate error, what if there was just some networking problem? And that was after I had just reviewed `FatalPredicateError` in the previous file.

What do you think about adding a `IsFatalPredicateError` and also a wrapper error that included the link to the documentation and then just calling `utils.FatalError`? Something along the lines of:

```go
func IsPredicateError(err error) bool {
    return strings.Contains(err.Error(), "predicate expression")
}

type PredicateError struct {
   err error
}

func (p PredicateError) Error() string {
    return fmt.Sprintf("%s Check syntax at https://goteleport.com/docs/setup/reference/predicate-language/#resource-filtering", p.err.Error())
}

func (c *AppsCommand) ListApps(clt auth.ClientI) error {
...
    case err != nil:
        if IsPredicateError(err) {
            utils.FatalError(PredicateError{err})
        }
        return trace.Wrap(err)
```